Mappy/TypeMapGeneric.cs

```diff
@@ -76,11 +76,20 @@ internal TypeMap(Type idAttribute)
                 }
                 else if (isEnumerable)
                 {
-                    methodName = isComplex
-                        ? nameof(MappingContext.ConvertListComplex)
-                        : nameof(MappingContext.ConvertList);
-                }
-                else if (nullableType != null)
+                    if (propOrFieldType.GetGenericTypeDefinition() == typeof(SortedSet<>))
+                    {
+                        methodName = isComplex
+                            ? nameof(MappingContext.ConvertSortedSetComplex)
+                            : nameof(MappingContext.ConvertSortedSet);
+                    }
+                    else
+					{
```

Comment:
Please replace tabs with spaces to fix indentation

Reply:
Sure, I have not noticed this before.